__unported__/medical_emr/models/medical_patient_disease.py

```diff
@@ -21,82 +21,12 @@
 #
 ###############################################################################
 
-from openerp import models, fields, api
+from openerp import models, fields
 
 
 class MedicalPatientDisease(models.Model):
     _name = 'medical.patient.disease'
     _description = 'Medical Patient Disease'
 
-    @api.one
-    @api.depends('short_comment', 'pathology_id', 'pathology_id.name')
-    def _compute_name(self):
-        name = self.pathology_id.name
-        if self.short_comment:
-            name = '%s - %s' % (name, self.short_comment)
-        self.name = name
-
-    @api.one
-    @api.depends('active')
-    def _compute_expire_date(self):
-        if self.active:
-            self.expire_date = False
-        else:
-            self.expire_date = fields.Datetime.now()
-
-    @api.one
-    def action_invalidate(self):
-        self.active = False
-
-    @api.one
-    def action_revalidate(self):
-        self.active = True
-
-    name = fields.Char(compute='_compute_name', store=True)
-    treatment_description = fields.Char()
-    expire_date = fields.Datetime(compute='_compute_expire_date', store=True)
-    short_comment = fields.Char()
-    pathology_id = fields.Many2one(
-        comodel_name='medical.pathology', string='Pathology', index=True,
-        required=True)
-    physician_id = fields.Many2one(
-        comodel_name='medical.physician', string='Physician', index=True)
     pcs_code = fields.Many2one(
```

Comment:
Ahhh yes my bad, I was just waking up when I was reading all of this. I think I made another similar comment in one of the other PRs. 

No prob, I can take care of `medical.procedure`. They're basically the ICD-10 codes, they just need to be a Many2one because multiple codes can be related to multiple diseases. In fact, it may need to be a Many2many - I'm going to ask a friendly in the med field.